security/advancedtls/advancedtls_integration_test.go

```diff
@@ -1,3 +1,5 @@
+// +build go1.10
```

Comment:
Does this mean that the go1.9 functionality is verified in this integration test? Is it possible to not add this build constraint?

Reply:
Github can't seem to keep track of the association between comments and the corresponding file/line-number when you do a force commit. Unfortunate.

Reply:
But don't you want to verify the functionality for go1.9 as well?

Reply:
I'll add testing for go1.9 separately. Thanks for bringing it up!